### PR TITLE
Correction of docs for `lagInFrame` and `leadInFrame` functions

### LINKED ISSUE #65952: lagInFrame and leadInFrame functions missing offset argument.
**Company or project name**
n/a

**Describe the issue**
The `lagInFrame` and `leadInFrame` functions as listed in https://clickhouse.com/docs/en/sql-reference/window-functions#functions do not indicate that that they support an `offset` argument. This differs from what is shown for the `nth_value` function.

**Additional context**
n/a

### PR DESCRIPTION
**Pull request by jwoodhead**
Include offset argument for lagInFrame and leadInFrame window functions.
fixes #65952

### Changelog category (leave one):
- Documentation (changelog entry is not required)

### CODE REVIEW

File: docs/en/sql-reference/window-functions/index.md [hunk 1]
```diff
@@ -80,8 +80,8 @@ These functions can be used only as a window function.
 - `nth_value(x, offset)` - Return the first non-NULL value evaluated against the nth row (offset) in its ordered frame.
 - `rank()` - Rank the current row within its partition with gaps.
 - `dense_rank()` - Rank the current row within its partition without gaps.
-- `lagInFrame(x)` - Return a value evaluated at the row that is at a specified physical offset row before the current row within the ordered frame.
-- `leadInFrame(x)` - Return a value evaluated at the row that is offset rows after the current row within the ordered frame.
+- `lagInFrame(x, offset)` - Return a value evaluated at the row that is at a specified physical offset row before the current row within the ordered frame.
+- `leadInFrame(x, offset)` - Return a value evaluated at the row that is offset rows after the current row within the ordered frame.
```
Comment (yariks5s) [inline docs/en/sql-reference/window-functions/index.md:84]: Actually, `lag`/`leadInFrame` can take 1-3 arguments

**Comment by robot-clickhouse** [comment]
*This is an automated comment for commit ad6ddf634db8318f48b8f5e95d17473a1e5ae3e3 with description of existing statuses. It's updated for the latest CI running*

[✅ Click here](https://s3.amazonaws.com/clickhouse-test-reports/65998/ad6ddf634db8318f48b8f5e95d17473a1e5ae3e3/ci_running.html) to open a full report in a separate page

<details><summary>Successful checks</summary>
<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>Style check</td><td>Runs a set of checks to keep the code style clean. If some of tests failed, see the related log from the report</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/65998/ad6ddf634db8318f48b8f5e95d17473a1e5ae3e3/style_check.html">✅ success</a></td></tr>
<tbody>
</table>
</details>

File: docs/en/sql-reference/window-functions/index.md (hunk 1 above)
Comment (jwoodhead) [reply docs/en/sql-reference/window-functions/index.md:84]: Two questions then:
1. Is there a specific format used to indicate optional arguments?
2. What is the optional third argument? The description only mentioned an offset.

Comment (yariks5s) [reply docs/en/sql-reference/window-functions/index.md]: The third argument is a default value, used when the offset is outside the frame.

**Pull request merged**